userapp/src/components/user/UserForm.js

```diff
@@ -90,65 +109,76 @@ const UserForm = ({ initialUser, onSubmit}) => {
 
     return (
     <div>
-        <form className={userFormClasses.userForm} autoComplete="off" onSubmit={onSubmitForm}>
+        <form className={userFormClasses.userFormStyle} autoComplete="off" onSubmit={onSubmitForm}>
+            <Typography variant="h5" component="h2">{header}</Typography>
             <Grid container
-                direction="column"
-                justify="center"
-                alignItems="center">
+                direction="row"
+                justify="flex-start"
+                alignItems="flex-start"
+                spacing={1}>
                 <Grid item xs={6}>
                     <HelixTextField
```

Comment:
first name and last name should be simple to combine into one call as well

Reply:
Then, I will change the object on the bottom into Full Name instead